tooling/miniutil/src/build/terminator.rs

```diff
@@ -71,23 +72,24 @@ impl FunctionBuilder {
         }
     }
 
-    /// Calls a function that neither returns nor unwinds.
+    /// Calls a function that neither returns nor unwinds using Rust calling conventions.
```

Comment:
```suggestion
    /// Calls a function that neither returns nor unwinds using the Rust calling convention.
```
(same below a few more times)